dojo/unittests/tools/test_sarif_parser.py

```diff
@@ -297,25 +326,43 @@ def test_flawfinder(self):
             finding = findings[0]
             self.assertEqual("random/setstate:This function is not sufficiently random for security-related functions such as key and nonce creation (CWE-327).", finding.title)
             self.assertEqual("Medium", finding.severity)
-            self.assertEqual("This function is not sufficiently random for security-related functions such as key and nonce creation (CWE-327).", finding.description)
+            description = '''**Result message:** random/setstate:This function is not sufficiently random for security-related functions such as key and nonce creation (CWE-327).
+**Snippet:**       is.setstate(std::ios::failbit);
```

Comment:
Snippet can break the Markdown format.
We should use triple back-tick to escape snippet of code.

Reply:
Very good idea, done.